apps/studio/components/interfaces/Auth/index.tsx

```diff
@@ -1,5 +1,6 @@
+export { default as AdvancedAuthSettingsForm } from './AdvancedAuthSettingsForm/AdvancedAuthSettingsForm'
 export { default as AuthProvidersForm } from './AuthProvidersForm/AuthProvidersForm'
+export { default as BasicAuthSettingsForm } from './BasicAuthSettingsForm/BasicAuthSettingsForm'
 export { default as EmailTemplates } from './EmailTemplates/EmailTemplates'
 export { default as SmtpForm } from './SmtpForm/SmtpForm'
-export { default as BasicAuthSettingsForm } from './BasicAuthSettingsForm/BasicAuthSettingsForm'
-export { default as AdvancedAuthSettingsForm } from './AdvancedAuthSettingsForm/AdvancedAuthSettingsForm'
+export { ThirdPartyAuthForm } from './ThirdPartyAuthForm'
```

Comment:
avoid barrel files? import ThirdPartyAuthForm directly in each file

Reply:
This file is not barrel file. It doesn't reexports (imports and exports) any components. It exports a single component which uses components from sibling components.